sample.env

```diff
@@ -50,10 +50,15 @@ SB_RALLY_POINT_SERVERS={"local":{"desc":"Local","address":"::ffff:127.0.0.1","po
 #SB_ROUTE_CREATOR_PORT=14099
 
 
-# JSON string configuring the storage of uploaded files. Currently there is only
-# one option, storing to the filesystem. You can change the path (relative to
-# the server directory) to change where it writes to.
-SB_FILE_STORE={"filesystem":{"path":"server/uploaded_files"}}
+# A string denoting which file store implementation to use. Must match one of
+# the implementations below (either "filesystem" or "dospaces" for now).
+SB_FILE_STORE="filesystem"
+# JSON string configuring the storage of uploaded files. Currently there are
+# two options, storing to the filesystem and storing to the DigitalOcean Spaces.
+# For the filesystem option, you can change the path (relative to the server
+# directory) to change where it writes to. For the Spaces option, you *must* set
+# the URL to the correct endpoint, and matching access keys.
+SB_FILE_STORE_IMPLEMENTATIONS={"filesystem":{"path":"server/uploaded_files"},"dospaces":{"endpoint":"region.digitaloceanspaces.com","accessKeyId":"ACCESS_KEY_ID","secretAccessKey":"SUPER_SECRET_ACCESS_KEY","bucket":"shieldbattery"}}
```

Comment:
I don't really understand the point of having two config vars here, this one just contains a key (`dospaces`) that isn't even used for anything? Why not just keep the original key and if it contains a `filesystem` key, we store to the file system, and if it stores a `doSpaces` key, we store to Spaces? I think that was the original intent

If you're trying to give an example of the configuration for both, just include that in a commented out line